Makefile

```diff
@@ -49,7 +49,10 @@ binaries: clean \
 	_output/bin/kubectl.lima \
 	_output/share/lima/lima-guestagent.Linux-x86_64 \
 	_output/share/lima/lima-guestagent.Linux-aarch64 \
-	_output/share/lima/lima-guestagent.Linux-riscv64
+	_output/share/lima/lima-guestagent.Linux-riscv64 \
+	_output/share/lima/lima-guestagent.FreeBSD-x86_64 \
+	_output/share/lima/lima-guestagent.FreeBSD-aarch64 \
```

Comment:
The binary name should be `FreeBSD-arm64`, if `uname -m` returns `arm64`

Reply:
Then we would need special code for that, to make arch depend on os.

Reply:
Currently we just use the enum values directly, as in FREEBSD "-" AARCH64

Reply:
On the second thought `aarch64` is probably ok, as it is hard coded for the `.arch` field in the YAML and the `qemu-system-<ARCH>` binary name.